modules/update-other-images.adoc

```diff
@@ -35,4 +35,10 @@ $ oc -n stackrox set image ds/collector compliance=registry.redhat.io/advanced-c
 ----
```

Comment:
[Not blocking this change]

@IAmATeaPot418 @gaurav-nelson 
It looks like our instructions are missing upgrade steps for admission controller. I don't know if it can update itself.
https://docs.openshift.com/acs/3.67/upgrading/upgrade-roxctl.html#update-other-images_upgrade-roxctl

cc @misberner